### ISSUE TITLE
About Dataset

### ISSUE THREAD

**Issue by xnuohz** [mims-harvard/TDC#32]
What is the measurement of Y in Davis?
Thanks:)

**Comment by xnuohz** [comment]
btw, I think it's nice to define a function in `bi_pred_dataset.DataLoader` to get total sorted smiles list or protein sequences for each dataset.

**Comment by kexinhuang12345** [comment]
Hi, for Davis, it is Kd. What do you mean by sorted smiles list? Like all the smiles strings in the dataset? what would be the use case? thanks!

**Comment by xnuohz** [comment]
It may be more convenient to calculate the similarity matrix of drug smiles or target protein sequences.

**Comment by kexinhuang12345** [comment]
Yes, that makes sense. We will try to include it in our MolConverter class coming soon.
